lang/main.json

```diff
@@ -894,6 +894,7 @@
         "unavailableTitle": "Recording unavailable",
         "uploadToCloud": "Upload to the cloud"
     },
+    "screenshareDisplayName": "{{name}}'s screen",
```

Comment:
This PR only Internationalizes screenshare display name, @jallamsetty1 can you have your translators update the lang files?